### PR TITLE
ddl: do not switch to import mode during adding index

### LINKED ISSUE #48259: tidb bootstrap takes a long time
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)
1. deploy a cluster with 3 tidb nodes
2. import data about 25w partitions and 5w tables
3. restart tidb 

### 2. What did you expect to see? (Required)
tidb can restart immediately 
### 3. What did you see instead (Required)
it's takes a long time to bootstrap
图片

tidb-0
图片

tidb-1
图片

tidb-2 

图片

### 4. What is your TiDB version? (Required)
```sql
[10:55:48]TiDB root:INFORMATION_SCHEMA> select tidb_version();
+-----------------------------------------------------------+
| tidb_version()                                            |
+-----------------------------------------------------------+
| Release Version: v7.5.0-test                              |
| Edition: Community                                        |
| Git Commit Hash: fa75dbd363128183665c03e06af67ed9066c1d98 |
| Git Branch: heads/refs/tags/v7.5.0-test                   |
| UTC Build Time: 2023-11-02 06:43:24                       |
| GoVersion: go1.21.3                                       |
| Race Enabled: false                                       |
| Check Table Before Drop: false                            |
| Store: tikv
```

### PR DESCRIPTION
**Pull request by tangenta**
### What problem does this PR solve?

Issue Number: close #48259

Problem Summary: 

See https://github.com/pingcap/tidb/issues/48259#issuecomment-1801342245.

### What is changed and how it works?

- Do not execute `show config` in initialization of lightning env.
- Do not switch to import mode for raftkv2.

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  >   

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: pkg/ddl/ingest/backend_mgr.go [hunk 1]
```diff
@@ -62,11 +59,6 @@ func newLitBackendCtxMgr(ctx context.Context, sctx sessionctx.Context, path stri
 	if err != nil {
 		logutil.BgLogger().Warn("ingest backfill may not be available", zap.String("category", "ddl-ingest"), zap.Error(err))
 	}
-	isRaftKV2, err := util.IsRaftKv2(ctx, sctx)
```
Comment (Benjamin2037) [inline pkg/ddl/ingest/backend_mgr.go:65]: Why before we set up this check？for kv row format compatible？

Comment (tangenta) [reply pkg/ddl/ingest/backend_mgr.go:65]: This is used to check if multi-rocks is enabled.

**Review by GMHDBJD — APPROVED** [review]
LGTM

File: pkg/ddl/ingest/backend_mgr.go (hunk 1 above)
Comment (Benjamin2037) [reply pkg/ddl/ingest/backend_mgr.go:65]: OK

**Review by Benjamin2037 — APPROVED** [review]
LGTM

**Pull request merged**